dev/com.ibm.ws.wsoc/resources/com/ibm/ws/wsoc/internal/resources/WebSockets.nlsprops

```diff
@@ -228,8 +223,8 @@ missingslashornull.uri=CWWKH0052E: An exception occurred during deployment/start
 missingslashornull.uri.explanation=ServerEndpoint is invalid because Endpoint URI is either null or does not start with / in endpoint class. WebSocket application is not available for requests.
 missingslashornull.uri.useraction=Define non null Endpoint URI beginning with /. After you resolve the problem, deploy the application again.
 
-wsoc.feature.not.loaded.correctly=CWWKH0053E: The websocket feature did not load with an acceptable version.
-wsoc.feature.not.loaded.correctly.explanation=The websocket feature is loaded with a version that corresponds to the Java [Jakarta] EE specification of that websocket feature. The feature was loaded with no version defined.
+wsoc.feature.not.loaded.correctly=CWWKH0053E: The websocket container did not load with an acceptable version.
```

Comment:
```suggestion
wsoc.feature.not.loaded.correctly=CWWKH0053E: The WebSocket container did not load with an acceptable version.
```

Reply:
Also during an internet search, I noticed that websocket container is mentioned as **WebSocketContainer**.

Is that the case here as well? should it be mentioned together as one word?

Reply:
`WebSocketContainer` would refer to the actual class. Stating Websocket container should be fine.